pom.xml

```diff
@@ -38,6 +38,11 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-graphics</artifactId>
+            <version>20.0.1</version>
+        </dependency>
```

Comment:
What is this needed for?

Reply:
??

Reply:
it was needed before I rewrote the code using var. I deleted it now.